public/app/themes/justice/page_home.php

```diff
@@ -1,41 +1,22 @@
 <?php
 /**
+ * The template for displaying 401 pages
  *
- * Template name: Home
- * Template Post Type: page
  */
 
 get_header();
-
-?>
-
-    <div id="highlight-wrapper">
-        <div class="container-wrapper">
-            <div id="highlight">
-                <div class="tab-group">
-                    <!-- PAGE CONTENT -->
-                    <!-- ------------------------------------ -->
-                    <article>
-                        <a>
-                            <img src="<?php echo get_template_directory_uri() ?>/dist/img/scales-of-justice.jpg" alt="Scales of justice" width="474" height="285" />
-                        </a>
-                        <header>
-                            <?php the_content() ?>
-                        </header>
-                    </article>
-                    <!-- ------------------------------------ -->
-                    <!-- end/ PAGE CONTENT -->
-                </div>
-
-                <?php get_sidebar('right'); ?>
-
-            </div>
-        </div>
-    </div>
-    <div id="footer-bar">
-        <div>
-        </div>
-    </div>
-<?php
-
 get_footer();
+
+$templates = array( 'templates/home.html.twig');
```

Comment:
Hi Emily, I'm loving the work!
One observation: throughout the file, we can see [] arrays, can we use those here?